### ISSUE TITLE
UE 5.0.0 support

### ISSUE THREAD

**Issue by tobowers** [GDi4K/unreal-fetch#8]
Hello! I tried to compile again for UE5 but it doesn't seem to work. Any help or thoughts? I know it's in early access still, but this is a fantastic plugin that I'd love to use.

**Comment by soularmindmedia** [comment]
Yes, at the moment, I cannot compile my project in UE5 because of Fetch. 
It's a great plugin we need in UE5.

Thanks !

**Comment by Dhruv-Ray** [comment]
Is this still an issue?
Or has there been a fix yet?

**Comment by JulesVerny** [comment]
I wanted to try this utility, before I buy in marketplace. 

However it does not compile, build when copied this repository plugin into a Unreal 5.1 project.  So I do not know if I will pursue, if Unreal 5.x is not supported. 
It comes up with the following errors:
'Fetch' is Incompatible. The 'Fetch' plugin was designed for build 4.26.0. Attempt to load it anyway?
But when I hit yes it comes back with...
The following modules are missing or built with a different engine version: Fetch
Would you like to rebuild them now?
When I reply Yes, it comes back with. 
<project> could not be compiled. Try rebuilding from source manually. 
I have Visual studio open. but cannot build this.
